src/components/ShareMyLocationButton.js

```diff
@@ -19,6 +19,7 @@ const ShareMyLocationButton = () => {
     const lng = position.coords.longitude;
     const newMapCenter = { lat, lng };
     setUserCenterMap(newMapCenter);
+    setNewCenterMap({ ...newMapCenter, isDragged: false });
 
     if (!sharingLocation.current) {
       setNewCenterMap(newMapCenter);
```

Comment:
I'm not sure but I think you forgot to pass some value for `isDragged` property. Am I right?

Reply:
Looks like this value is set to true in the `setNewCenterMap({ ...defaultCenterMap, isDragged: true });` method below.

Reply:
It's possible. Even though this time the setter method is inside a if condition, but I could be wrong.

Reply:
@alxmcr , nothing is missing in that line of code

Reply:
I have the same question as Alejandro. By not specifying `isDragged` explicitly, aren’t you implicitly passing in `undefined` as the value, which is “falsey” and behaves as if you passed in `false`? It would be more clear to give it an explicit value in all cases, I think.

Reply:
I believe it's much better if we pass an explicit value 👍  because there are places where we don't pass it like the `FlyToMe` button and it's possible there are others places 👀 .

Reply:
@segdeha, I don't understand what exactly you want me to change and where, would be grateful of you give an example

Reply:
I think it would just be a matter of making line 25 match line 22. So, changing `setNewCenterMap(newMapCenter);` to `setNewCenterMap({ ...newMapCenter, isDragged: false });`

Sorry for not making my request more concrete before!

Reply:
@segdeha, got it - since the highlighted line was line 22, I couldn't understand what both of you were referring to. Line 25 has been updated.